deployment/run.sh

```diff
@@ -30,11 +30,27 @@ case "${mode}" in
   install)
     install_logging
     ;;
+  uninstall)
+    delete_logging
+    ;;
+  reinstall)
+    delete_logging
+    install_logging
+    ;;
   migrate)
```

Comment:
on second thought, given that upgrade) only does the migration _if_ the admin-cert doesn't exist I think we should keep it separate so someone can explicitly make a migrate) call if needed...